pkg/dashboard/uiserver/embedded_assets_rewriter.go

```diff
@@ -17,19 +17,30 @@ package uiserver
 import (
 	"net/http"
 	"os"
+	"path"
 	"sync"
 	"time"
 
+	"github.com/pingcap/log"
 	"github.com/pingcap/tidb-dashboard/pkg/config"
 	"github.com/pingcap/tidb-dashboard/pkg/uiserver"
+	"go.uber.org/zap"
 )
 
 var once sync.Once
 
 // Assets returns the Assets FileSystem of the dashboard UI
 func Assets(cfg *config.Config) http.FileSystem {
 	once.Do(func() {
-		uiserver.RewriteAssets(assets, cfg, func(fs http.FileSystem, f http.File, path, newContent string, bs []byte) {
+
+		exePath, err := os.Executable()
+		if err != nil {
+			log.Fatal("Failed to get executable path", zap.Error(err))
+		}
+
+		distroResFolderPath := path.Join(path.Dir(exePath), "distro-res")
```

Comment:
These logics are duplicated with `dashboard.go`. Could you have some refinements?

Reply:
done!